.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+Click the `Preview` tab and select a PR template:
+
+- [Bug fix](?expand=1&template=bugfix.md)
```

Comment:
Do these actually show up as links?

Reply:
To be sure, try
https://github.com/kenji-miyake/test-pr-template/compare/main...kenji-miyake-patch-1